### ISSUE TITLE
ImportError: cannot import name 'fast_to_sql' from 'fast_to_SQL'

### ISSUE THREAD

**Issue by alekks** [jdglaser/fast-to-sql#3]
Apologies if I'm doing something obviously wrong, but just started using python and not 100% about the more indepth workings of modules etc.

Traceback (most recent call last):
```
File "C:\Users\usr\github\pyAutoReport\importDailyPendingToDB.py", line 1, in <module>
    from fast_to_SQL import fast_to_sql as fts
ImportError: cannot import name 'fast_to_sql' from 'fast_to_SQL' (C:\Users\usr\AppData\Local\Programs\Python\Python38-32\lib\site-packages\fast_to_SQL\__init__.py)
```

`__init__.py` is empty

**Comment by hthiel** [comment]
Hi Alekks,

I would guess that this is just a typo in the package readme. The root directory of your package (as you can verify from your error message) is `...\fast_to_SQL\`. This is what you refer to when doing `import fast_to_SQL` or `form fast_to_SQL import ...`. But the name of the submodule that has all the functionality in it is `fast_to_SQL.py` and not `fast_to_sql.py`.  

If you use `from fast_to_SQL import fast_to_SQL as fts`, you should be able to use `fts.to_sql_fast()` as expected...at least that works for me.

the `__init__.py` is empty most of the times. Python requires that a (sub)directory contains an `__init__.py` file before it can be recognized as a (sub)package.  If you (as a package author) want import other stuff in the background (mostly from modules within the same package) to be available at the top level, you can do it there (it's rare for smaller packages, I would say, though). But you don't have to. `__init__.py` is alway executed when loading the corresponding package level.  In the error you shared, the import system is just expecting to find a definition of what `fast_to_sql` is there, if it cannot find `fast_to_sql.py` in the directory.

So don't worry about "doing something obviously wrong", it's rather easy once you get the trick ;)

Best

**Comment by jjr5330** [comment]
I agree the read me is wrong it should be **from fast_to_SQL import fast_to_SQL as fts**

**Comment by ibuda** [comment]
I confirm that @jjr5330 proposal solves the problem.

**Comment by jdglaser** [comment]
Fixed in new package release [2.1](https://github.com/jdglaser/fast-to-sql/releases/tag/v2.1). Closing.
